src/platform/nftPlatforms.ts

```diff
@@ -82,11 +82,17 @@ export const NFT_PLATFORM_CONFIG: { [key: string]: NFTPlatform } = {
 };
 
 // Function to detect NFT details from URL
-export function detectNFTDetails(url: string): NFTExtraction | undefined {
+export async function detectNFTDetails(url: string): Promise<NFTExtraction | undefined> {
```

Comment:
It would be great to update tests to keep the coverage of this function.

Reply:
Updated tests for async functions